multi-agent/lib/mas/core/tracing/service.py

```
@@ -0,0 +1,67 @@
+from __future__ import annotations
+
+from abc import ABC, abstractmethod
+from contextlib import contextmanager
+from typing import Any, Dict, Iterator, List, Optional
```

Ruff (0.15.21)

[warning] Import from `collections.abc` instead: `Iterator`

Import from `collections.abc`

(UP035)